resources/js/router/views/DashboardAdmin/index.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+	<div class="container justify-content-center">
+		<div class="text-left text-primary px-3">
```

Comment:
You don't need the class of `text-left` here, because the content is already set to `text-align:left` by default 👍